src/booking-list/components/BookingDetails.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { datesManager } from 'src/booking-list/datesManager';
+import { Loading } from 'src/commonComponents/Loading';
+import { useLazyGetBookingsIdQuery } from 'src/services/hondaApi';
+import { ButtonsBar } from './ButtonsBar';
+
+export const BookingDetails = () => {
+  const params = useParams();
+
+  const [trigger, result] = useLazyGetBookingsIdQuery();
+
+  const idParams = params.bookingId;
+
+  const [rideCompletionText, setRideCompletionText] = useState('');
+
+  useEffect(() => {
+    if (idParams) {
+      const parsedParams = idParams.split('$');
+      const username = parsedParams[0];
+      const carId = parsedParams[1];
+      const timeSec = parsedParams[2];
+
+      trigger({ username, carId, startTime: timeSec });
+
+      if (result.isSuccess) {
+        setRideCompletionText(
+          result.currentData.booking.bookingOwner.settings.rideCompletionText,
+        );
+      }
+    }
+  }, [idParams, result.currentData, result.isSuccess, trigger]);
+
+  return (
+    <div className={'bookingPage'}>
+      {result.isLoading ? (
+        <Loading />
+      ) : (
+        <>
+          <div className={'bookingWrapper'}>
+            {result.isSuccess && (
+              <table>
+                <caption className={'bookingHeader cellDecoration'}>
+                  Booking details
+                </caption>
+                <tbody>
+                  <tr>
+                    <td className={'cellDecoration'}>Инициатор поездки</td>
+                    <td className={'cellDecoration'}>
+                      {result.currentData.booking.bookingOwner.firstName}
+                    </td>
+                  </tr>
+                  <tr>
+                    <td className={'cellDecoration'}>Автомобиль</td>
+                    <td className={'cellDecoration'}>
+                      {result.currentData.booking.bookingOwner.availableCars}
```

Comment:
booking object itself contains information about the car used for the booking. You should get the carId from booking, and not from the user, because the user might have several cars

Reply:
changed